### PR TITLE
docs: alternative clients for MUD

### PR DESCRIPTION
**Pull request by qbzzt**
Replacing #1469 (which developed irreconcilable merge conflicts)

### CODE REVIEW

File: docs/pages/_meta.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
   "quick-start": "Quickstart",
   store: "Store",
   world: "World",
-  "client-side": "MUD client",
+  client: "MUD clients",
```
Comment (holic) [inline docs/pages/_meta.js:10]:
```suggestion
  clients: "MUD clients",
```

File: docs/pages/client/client-typescript.mdx [hunk 2]
```diff
@@ -1,10 +1,4 @@
-# Interacting with MUD from a client
-
-MUD is client agnostic: any client -- a browser, a game engine, or an iOS app -- can implement the synchronization protocol and a client-side cache to replicate [Store](/store) tables, along with the necessary infrastructure to send transactions to the World.
-
-However, currently we **only** officially support the browser and Node. We provide tooling for representing the onchain state and replicate it from an RPC or [the Indexer](/indexer) via a network stack.
-
-There also exists tooling for [Unity](https://github.com/emergenceland/unimud) from the folks at [emergence.land](https://emergence.land).
+# Typescript Client
```
Comment (holic) [inline docs/pages/client/client-typescript.mdx:1]:
```suggestion
# Typescript client
```

File: docs/pages/_meta.js (hunk 1 above)
Comment (qbzzt) [reply docs/pages/_meta.js]: Done, along with appropriate renames.

**Comment by qbzzt** [comment]
Make unnecessary by https://github.com/latticexyz/mud/pull/1915 and https://github.com/latticexyz/mud/pull/1914

**Pull request closed without merging**